step-strikers-app/step-strikers-app/Menu Screens/BattleMenuViewController.swift

```diff
@@ -44,11 +44,23 @@ class BattleMenuViewController: UIViewController {
     }
     
     @objc func createPressed() {
```

Comment:
i think we should indicate the senders of these methods are buttons so.
 @objc func createPressed(_ sender:UIButton!)
and
@objc func joinPressed(_ sender:UIButton!)

You shouldn't have to change the code where you select this function as the target when the button is pressed.